compute_tools/src/config.rs

```diff
@@ -79,6 +79,14 @@ pub fn write_postgres_conf(
         ComputeMode::Replica => {
             // hot_standby is 'on' by default, but let's be explicit
             writeln!(file, "hot_standby=on")?;
+
+            // Inform the replica about the primary state
+            // Default is 'false'
+            writeln!(
+                file,
+                "neon.primary_is_running={}",
+                spec.primary_is_running.unwrap_or(false)
+            )?;
```

Comment:
Are you sure that it is safe default?
If we assume that primary is running but it is not, then startup of hot-standby replica hangs forever.
That is what happening now with e2e tests.
May be we should assume that if information about primary is not available, then it is not active?
In this case this PR should pass e2e tests even without changes in control plane?

Reply:
By "safe", I meant "you won't get incorrect query results". You're right that hanging forever is not nice either, and may even be worse. Still, I'd be more comfortable waiting than possibly returning wrong results.

Reply:
Well, I agree with you.
I just want to postpone this change until https://github.com/neondatabase/cloud/pull/10183 is merged.
It gives a chance to e2e tests to pass.

Reply:
Fair. Alternatively, you could merge https://github.com/neondatabase/cloud/pull/10183 first. That one passes the tests without this PR. I'd prefer to commit this in one piece, otherwise it's hard to track what issues remain.

Do we need any special considerations in how this is rolled out to production? We need to make sure the console change are deployed first?

Reply:
So https://github.com/neondatabase/cloud/pull/10183 is now merged.
What we should do then?
It seems to me that it is still "safer" to keep default value of primary_is_running=false, because it corresponds to the current status quo: when we start any compute mode, we assume that it happens after normal shutdown.
It also simplify rollup: with default=false we can deploy control plane and compute in any order:
Control plain includes "primary_is_running" is spec, but it will not be transformed into GUC until compute is deployed. And visa versa - if we deploy first compute, then with default value `primary_is_running=false`
it preserves the current behaviour.

We can always change default in future.